netbox/ipam/views.py

```diff
@@ -52,7 +47,7 @@ def get_extra_context(self, request, instance):
         )
 
         return {
-            'related_models': related_models,
+            'related_models': self.get_related_models(request, instance),
```

Comment:
This returns two entries for "Interfaces," because we're querying both device and VM interfaces. The simplest solution would be to exclude both, which IMO is reasonable.